admin-users/src/test/kotlin/com/epam/drill/admin/users/UserAuthenticationTests.kt

```diff
@@ -125,4 +128,9 @@ class UserAuthenticationTest {
         }
         bind<UserAuthenticationRoutes>() with eagerSingleton { UserAuthenticationRoutes(di) }
     }
-}
\ No newline at end of file
+
+    private fun TestApplicationRequest.addBasicAuth(username: String, password: String) {
```

Comment:
case for implementing separate function -> is it better be inlined?

Reply:
explain _what_ do we test here / what is stubbed and what is not
_somehow_:
-comments?
-test names?
-test class name?